tests/test_npm_package_contents.py

```
@@ -137,6 +137,91 @@ def compare_expected_vs_actual(expected_files: Set[str], actual_files: Set[str])
+            # Match: COPY path/to/source ./dest OR COPY path/to/source .
+            for line in content.split('\n'):
+                line = line.strip()
+                if line.startswith('COPY ') and not line.startswith('COPY --from'):
+                    # Extract source path (first argument after COPY)
+                    parts = line.split()
+                    if len(parts) >= 3:
+                        source = parts[1]
+                        # Normalize: remove trailing slashes, handle directories
+                        source = source.rstrip('/')
+                        copy_sources.add(source)
+        except Exception as e:
```

Ruff (0.14.10)

Do not catch blind exception: `Exception`

(BLE001)